locales/app.yml

```
@@ -143,658 +165,764 @@ _version: 2
   fr: "Aucun répertoire Vagrant n'est spécifié dans le fichier de configuration"
   zh_CN: "尚未在配置中指定 Vagrant 目录"
   zh_TW: "尚未在設定檔中指定 Vagrant 資料夾"
+  de: "In der Konfigurationsdatei wurden keine Vagrant-Verzeichnisse angegeben"
 "Vagrant boxes":
   en: "Vagrant boxes"
   es: "Cajas Vagrant"
   fr: "Boîtes Vagrant"
   zh_CN: "Vagrant 容器"
   zh_TW: "Vagrant 容器"
+  de: "Vagrant-Boxen"
 "No outdated boxes":
   en: "No outdated boxes"
   es: "Sin cajas obsoletas"
   fr: "Aucune boîte obsolète"
   zh_CN: "没有需要更新的容器"
   zh_TW: "未有需要更新的容器"
+  de: "Keine veralteten Boxen"
 "Summary":
   en: "Summary"
   es: "Resumen"
   fr: "Résumé"
   zh_CN: "结果"
   zh_TW: "結果"
+  de: "Zusammenfassung"
 "Topgrade finished with errors":
   en: "Topgrade finished with errors"
   es: "Topgrade finalizado con errores"
   fr: "Topgrade terminé avec des erreurs"
   zh_CN: "Topgrade 运行完成，但存在错误"
   zh_TW: "Topgrade 執行部分成功"
+  de: "Topgrade wurde mit Fehlern beendet"
 "Topgrade finished successfully":
   en: "Topgrade finished successfully"
   es: "Topgrade finalizó exitosamente"
   fr: "Topgrade terminé avec succès"
   zh_CN: "Topgrade 运行成功"
   zh_TW: "Topgrade 執行成功"
+  de: "Topgrade wurde erfolgreich beendet"
 "Topgrade {version_str} Breaking Changes":
   en: "Topgrade %{version_str} Breaking Changes"
   es: "Topgrade %{version_str} Cambios Importantes"
   fr: "Topgrade %{version_str} Changements Cassants"
   zh_CN: "Topgrade %{version_str} 破坏性更改"
   zh_TW: "Topgrade %{version_str} 重大更改"
+  de: "Topgrade %{version_str} Breaking Changes"
 "Path {path} expanded to {expanded}":
   en: "Path %{path} expanded to %{expanded}"
   es: "Ruta %{path} expandida a %{expanded}"
   fr: "Le chemin %{path} a été transformé en %{expanded}"
   zh_CN: "已扩展 %{path} 至 %{expanded}"
   zh_TW: "已擴展 %{path} 至 %{expanded}"
+  de: "Pfad %{path} erweitert zu %{expanded}"
 "Path {path} doesn't exist":
   en: "Path %{path} doesn't exist"
   es: "La ruta %{path} no existe"
   fr: "Le chemin %{path} n'existe pas"
   zh_CN: "路径 %{path} 不存在"
   zh_TW: "路徑 %{path} 不存在"
+  de: "Pfad %{path} existiert nicht"
 "Cannot find {binary_name} in PATH":
   en: "Cannot find %{binary_name} in PATH"
   es: "No se pudo encontrar %{binary_name} en PATH"
   fr: "Impossible de trouver %{binary_name} dans le PATH"
   zh_CN: "在 $PATH 中找不到 %{binary_name} 二进制"
   zh_TW: "在 $PATH 中找不到 %{binary_name} 執行檔"
+  de: "Kann %{binary_name} nicht im PATH finden"
 "Failed to get a UTF-8 encoded hostname":
   en: "Failed to get a UTF-8 encoded hostname"
   es: "Error al obtener un nombre de host codificado en UTF-8"
   fr: "Échec de l'obtention d'un nom d'hôte encodé en UTF-8"
   zh_CN: "无法获取 UTF-8 编码的主机名称"
   zh_TW: "無法取得 UTF-8 編碼的主機名稱"
+  de: "Fehler beim Abrufen eines UTF-8-kodierten Hostnamens"
 "Failed to get hostname: {err}":
   en: "Failed to get hostname: %{err}"
   es: "Error al obtener el nombre del host: %{err}"
   fr: "Échec de l'obtention d'un nom d'hôte: %{err}"
   zh_CN: "无法获取主机名称：{err}"
```

Not your fault, but this needs a fix:)

```suggestion
  zh_CN: "无法获取主机名称：%{err}"
```

---

I will commit this one